spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -74,12 +75,12 @@ object SpdxLicenseAliasMapping {
             "ruby" to RUBY,
             "wtfpl" to WTFPL,
             "zlib" to ZLIB
-    )
+    ).mapValues { (_, v) -> v.toExpression() as SpdxExpression }
```

Comment:
Why is `as SpdxExpression` required?

Reply:
Thanks, that was indeed a left-over from an in-between state of the code.